TempleApp/src/main/java/example/TempleApp/Activities/ReadAllData.java

```diff
@@ -163,33 +163,33 @@ protected Void doInBackground(Void... params) {
 
                                 JSONObject innerObject = array.getJSONObject(jIndex);
 
-                                String id = innerObject.getString("ID");
-                                String name = innerObject.getString("NAME");
+                                String id = innerObject.getString(getString(R.string.ID));
+                                String name = innerObject.getString(getString(R.string.name));
 
                                 String[] str = name.split(getResources().getString(R.string.empty));
 
                                 if (flag == 1) {
-                                    if (id.substring(0, 3).equals("DON") && str[2].equals("PAID")) {
+                                    if (id.substring(0, 3).equals(getString(R.string.DON)) && str[2].equals(getString(R.string.PAID))) {
                                         model.setName(name);
                                         model.setCountry(id.substring(3, id.length()));
                                         list.add(model);
                                     }
                                 } else if (flag == 2) {
-                                    if (id.substring(0, 3).equals("DON") && str[2].equals("NOT PAID")) {
+                                    if (id.substring(0, 3).equals(getString(R.string.DON)) && str[2].equals(getString(R.string.NOT_PAID))) {
                                         model.setName(name);
                                         model.setCountry(id.substring(3, id.length()));
                                         list.add(model);
 
                                     }
                                 } else if (flag == 3) {
-                                    if (id.substring(0, 3).equals("REG") && str[2].equals("PAID")) {
+                                    if (id.substring(0, 3).equals(getString(R.string.REG)) && str[2].equals(getString(R.string.PAID))) {
                                         model.setName(name);
                                         model.setCountry(id.substring(3, id.length()));
                                         list.add(model);
 
                                     }
                                 } else if (flag == 4) {
-                                    if (id.substring(0, 3).equals("DON") && str[2].equals("NOT PAID")) {
+                                    if (id.substring(0, 3).equals(getString(R.string.DON)) && str[2].equals(getString(R.string.NOT_PAID))) {
```

Comment:
here it is `REG` I unexpedtly written DON please change that

Reply:
Changed it to REG